### PR TITLE
Update changelog and mark Camus as deprecated.

### PR DESCRIPTION
**Pull request by ewencp**

### CODE REVIEW

**Comment by ewencp** [comment]
Updated version of #38 against the right branch since new branches were created.

@Ishiihara Should be a quick review. @josephadler if you want to check the deprecation wording.

File: docs/intro.rst [hunk 1]
```diff
@@ -190,6 +198,11 @@ Requirements
 Contribute
 ----------
 
+.. warning::
+
+   Because Camus is deprecated both in Confluent Platform and upstream, only critical bug fixes are accepted to these
```
Comment (Ishiihara) [inline docs/intro.rst:203]: We cannot control the upstream repository, right?

**Comment by Ishiihara** [comment]
@ewencp Left a comment. Otherwise LGTM.

File: docs/intro.rst (hunk 1 above)
Comment (ewencp) [inline docs/intro.rst:203]: Yeah, but we know they won't accept patches anymore (really at all, although I think for something really bad we could get someone to listen). I just wanted to emphasize that all avenues for updates are basically closed at this point.

Comment (Ishiihara) [inline docs/intro.rst:203]: That makes sense. LGTM.

**Pull request merged**